src/playlist-maker.js

```diff
@@ -22,144 +30,175 @@ var indexInSources = function(arr, src) {
   return -1;
 };
 
-// factory method to return a new playlist with the following API
```

Comment:
Yeah, though, when working with it, I found it super useful to have that comment in there.